src/steps/vulnerabilities/__recordings__/fetchVulnerabilities_3247133843/recording.har

```diff
@@ -0,0 +1,180 @@
19236f79f77a3e0b182e482ab834179710feab599bb471d7254c5b3e1a0d9a16023b7b91033037753af076d40f6c1d538b93b197c9233eaf17659da702554e734ebe18a134d296a9ac0058f9c3aaec4109ab86fc5496845b27ce09153352f184212637a9c8456240903884aa57f3184247fe971125a916caf212a75588421249192c7896b459396cdc10897347646e42d169c84563481853e0996425473ce0732226fb1e024b422f216e93a0acd88bcc58293d08ac85bb0b4b7a0b9ff816f9e3cd8d33425861095f6164419117d9211d933769acad35c272b4f3184c85b2c38710b243a19d5c952286845e42d4e724abbdf8ac85b24af9a402ba2299c2c0300291b89288f13330b0c2159443c4e421489b14354cad8712d9ee4057a9ce881b9a1f1817cc54988228998202a555e038e3d928499c7896b45e303212a6516184262ec1e27a115890f84a854c48421243ed0e324b4228998202a15316108c914f63809ad48f65710958a983084680a272f7c724393f3f638f1174843fc0151e9294c934d85a8548a044388a6f0094d46388c8128a2299cbe816488ee7fac3809b3209ac24b154822b62059af105ba476f860e424f1161e27ae154d891c44a5620b0c21714c1e27be1321aa53479a375e8e87042f8955789c55a91df29ec5f55f7cec874dc6710931d73c4a3fa3944c1cf2f01b21ffd5873c1cd3fab1cd47b899871d6733200ecf402da23339cc537618a4cd381f67310a3babfc8037ff947ee6c31ffdc3bdeb52b2c33e83861dcc11d8e811ad1d8c6b0742c63e73e613bf847de61ce857679fc159eab3962b38059d7bd0f9646c5222433f1493f5bd9ad05246cc859cf1e996d5e2e957faad6bd1c3e73ef29990ae99716e2c7a334ed27d68b2cf74e623bfe4339d03fdea9f29d063e597cf384bfbf17d7eb8ba9de7cf839b1e47662090f3b870277002f7f1ddaf1f6fef9aebf693e3120ab649128d443324c50bd959515595d195a84ccbaa2ec72fde28d616a5958cd51c5d93c05fc918ef38ef2c6f5a2deac8011c28b9f8524a7ea16fdd7076bdb8cc98a128b2d9702292301d08d60b89a64927778b032b93caf2bc20d8b26d38bb3a2d2308c2bd0d272209adac0832492af33851491485e24e52bc50dc8d5868612eb4f20d27a2135f09bd2f96e471a29228ae74aa0cf7a0a3d721304252e440369c5d9d7486ba028299bbe14424214b47b0ffd499c7d99564e061970646175ac4861391c4584191d631d923d051d66e399442665589bd48995bd7f44e981b21fedb4def460d4a7b5b60491841d3a8f49cd9818d992d0c62dc0255064be17ca0e9dd7ef4352acde4c1cebc3053de3b63248bbec6f31ef725d1d739d017445f67b52f0c51341ea3af77dfd0d6f0dbf470b5b41bbc7fd6ea2e1c867dfdf1fbc3d72fd7e3972f7b5158c3aab634b2ac6d5328b4a5ac1b5956352b3bc9cb4ed7c2545648dbb45d5db41af5d8d6a26f14ab8c119dd275896e9d7bcd2a8b2c3714977e369c75de06ae0eb8110b13d685de68c3394adaf310dab21c578ef73c04cbdf337693f31b6e221ea2fc73fcfc9cc875f9eb1934ae455e8c62d65386cd6e9f810aec90f57c18b289d9418de0ceecc1cd745153ccc24eb69f7b96a1e5e10112a4c80a74c3ccd42034cae60e8530a17db46b8ae9ccd5d9dc5e1f4c9078f7961fb0c99be4c26343e625ce7ce897788973a05fee25cefb9eff8c97604dad1aa64aae4b63ab0ac47d0c5cadbcd47985bf96701f95d070172d9a5f99b6ea54de74b606cb6f55cbaeeee4ffda4be8b497d06f5e22d42df7cd4b1cdb65fba26a0a2f11cee4b88d78209373fbe1fdcfb7b73fc5323970079db045d916ada80517e8685d993a97555dd48da31a54b946f450621bd4e2e24dd536a2b515e87dc061a6abaedef70ed81f308aabe41b4e749f6196c6081706116e9fb102adb25c73ed2757499098c61ba110e571225a6104c1de7379e22891811b41c1d5bae1c4de9ea6b8280a492bceeedb73f1a025c81a6d38e99da7b639a2aadd9da78b2bf31bc16f546ce7f95a71e55f000000ffff\",\"1a6d57126e0c0fe176656c2d000000ffff0300aefd3666e1810000\"]"
```

Comment:
I know you called out the fact that this was gzipped in the PR description.  Is that a requirement specific to this integration?  Or was it a size concern?  I think typically we've leveraged the `unzipGzippedRecordingEntry` from @jupiterone/integration-sdk-testing to record the unzipped data.  (Example from an integration Stefan did a while back here:  https://github.com/JupiterOne/graph-datadog/blob/main/test/recording.ts#L14 )

Reply:
@adam-in-ict Thank you! I honestly didn't know this existed. I think the unzipped version will be about 30KB -> 100KB so a little bit bigger, but I think it should be okay.